dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceServiceTest.java

```diff
@@ -342,4 +342,25 @@ void testTrackedEntityInstanceGridWithNoDisplayAttributes()
         assertThrows( IllegalArgumentException.class,
             () -> entityInstanceService.getTrackedEntityInstancesGrid( params ) );
     }
+
+    @Test
+    void shouldCountOneEntityWhenOnePresent()
+    {
+        entityInstanceA1.setTrackedEntityType( trackedEntityTypeA );
+        entityInstanceService.addTrackedEntityInstance( entityInstanceA1 );
+
+        TrackedEntityInstanceQueryParams params = new TrackedEntityInstanceQueryParams();
```

Comment:
`param` can be inlined as its only used once. This makes the tests even more readable 😄 Same below.